frontend/src/components/Secrets/GSelectSecret.vue

```diff
@@ -120,6 +120,14 @@ export default {
     registerVuelidateAs: {
       type: String,
     },
+    filterSecretNames: {
```

Comment:
The name filterSecretName suggests that it is a function that filters a list of secretNames. In reality, however, it is an allowlist of possible or permitted secret names. The name should reflect that. I would use something like `allowedSecretNames` oder `permittedSecretNames`.

Reply:
changed to allowedSecretNames